app/lib/features/events/pages/create_edit_event_page.dart

```diff
@@ -304,33 +304,29 @@ class CreateEditEventPageConsumerState
       firstDate: DateTime.now(),
       lastDate: DateTime.now().addYears(1),
     );
-    if (date != null) {
-      if (isStartDate) {
-        _selectedStartDate = date;
-        _startDateController.text = eventDateFormat(date);
-        // if end date is empty and if start date is same or after end date
-        if (_endDateController.text.isEmpty ||
-            date.isSameOrAfter(_selectedEndDate)) {
-          _selectedEndDate = date;
-          _endDateController.text = eventDateFormat(date);
-        }
+    if (date == null) return;
+    if (!context.mounted) return;
```

Comment:
let's combine for compatness in reading:
```suggestion
    if (date == null || !context.mounted) return;
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1580/commits/37ecf0ea45e677c1578c8e65c3d87e73b94dc31b